test/remote/gateways/remote_pac_net_raven_test.rb

```diff
@@ -156,15 +156,15 @@ def test_authorize_and_void
   def test_authorize_capture_and_void
     auth = @gateway.authorize(@amount, @credit_card, @options)
     assert capture = @gateway.capture(@amount, auth.authorization)
-    assert void = @gateway.void(capture.authorization, {:pymt_type =>  capture.params['PymtType']})
-    assert_failure void
+    assert void = @gateway.void(capture.authorization, {:pymt_type => capture.params['PymtType']})
```

Comment:
Can you tweak the integration to return an authorization that includes the authorization code + the payment type, if that's required to be able to void? The expectation is that calling `gateway.void(authorization)` is all you need to void something.

The Authorize.Net adapter does this by joining things together with `#`: https://github.com/activemerchant/active_merchant/blob/93251aaf947361f3cab3be4d2356ecc636d928c7/lib/active_merchant/billing/gateways/authorize_net.rb#L736-L743